qa/L0_perf_analyzer/test.sh

```diff
@@ -120,6 +120,9 @@ for INPUT in INPUT0 INPUT1; do
     done
 done
 
+# Fix container installation of Open MPI
```

Comment:
What happens without this commit?

Reply:
We switched to the LD_LIBRARY_PATH solution 👍